tools/nl/embeddings/data/curated_input/sheets_svs.csv

```diff
@@ -1571,3 +1571,8 @@ Exports_EconomicActivity_ElectronicComponents,Electronics Components Exports,,,e
 Imports_EconomicActivity_ElectronicComponents,Electronics Components Imports,,,import of electronics;electronic components import; imports of electronic components;
 Exports_EconomicActivity_Pharmaceutical,Pharmaceutical Exports,,,export of pharmaceuicals;pharmaceutical export; exports of pharmaceuticals;
 Imports_EconomicActivity_Pharmaceutical,Pharmaceutical Imports,,,import of pharmaceuicals;pharmaceutical import; imports of pharmaceuticals;
+dc/topic/Veterans,Veterans,Veterans,,
```

Comment:
Could you add some alternate descriptions like how one would query for this in NL? To the Curated_Alternatives column

Reply:
done